changelogs/fragments/77969-apt-preferences.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - apt - fix package selection to include /etc/apt/preferences(.d)
```

This is a bugfix and should be backported to 2.13.

```suggestion
bugfixes:
  - apt - fix package selection to include /etc/apt/preferences(.d) (https://github.com/ansible/ansible/issues/77969)
```